engine/src/hopeit/app/api.py

```diff
@@ -183,4 +209,10 @@ def event_api(title: Optional[str] = None,
             }
         )
     """
-    return partial(_event_api, title, payload, query_args, responses)
+
+    if title is not None:
+        warnings.warn("title parameter is deprectated, use summary instead", DeprecationWarning)
```

Comment:
Typo: deprecated
Include version: is deprecated since 0.1.4 and will be removed in version 0.2.0

Create a file in docs lusting future deprecations:
Module
Description
Since version
Remove in version

Reply:
Check https://github.com/hopeit-git/hopeit.engine/blob/87644104a92e09478f4b356fb9507a83831d19ce/docs/source/deprecations.rst starting doc for deprecations.